<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="Measurement">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Measurement</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButton_ref">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Store Reference Measurement</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="canvas_blanc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="canvas_sweep">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>281</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>571</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>390</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Intensity Sweep</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_reset">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset All</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N of measurement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pausing time [s]</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_correctN">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Nonlinearity Correction</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_wl">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>200</number>
      </property>
      <property name="maximum">
       <number>900</number>
      </property>
      <property name="value">
       <number>505</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_correctE">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Electronic dark Correction</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="intSpinBox_averageScans">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_IT2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wavelength [nm]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_IT1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_init">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Init Devices</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Integration time I [ms]</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_pause">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.050000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_useTTLshutter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>using TTL-Shutter</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_correctLighting">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient Light Correction</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Integration time II [ms]</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="canvas_reac">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_FitPoints">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>390</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LinRegr">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>390</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>linear regression</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>440</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>slope</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>390</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Points to fit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>440</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCDNumber_slope">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>420</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCDNumber_slope_error">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>420</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>300</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse to Directory</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dummy</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>431</width>
        <height>401</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>427</width>
         <height>397</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>421</width>
          <height>401</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>time [s]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Blank [cts]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reaction [cts]</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_save">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save data to *.txt</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_browse">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>320</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
